app/assets/javascripts/admin/addon/templates/user-index.hbs

```diff
@@ -641,6 +641,17 @@
     <div class="field">{{i18n "user.invited.days_visited"}}</div>
     <div class="value">{{html-safe model.days_visited}}</div>
   </div>
+  <div class="display-row post-edits-count">
+    <div class="field">{{i18n "user.invited.post_edits_count" }}</div>
+    <div class="value">
+      {{html-safe model.post_edits_count}}
```

Comment:
Does this field have HTML? I suspect it's just a number and we can do `{{model.post_edits_count}}`